src/js/pm-image-editor.js

```diff
@@ -444,6 +448,11 @@
                     $scope.$broadcast('updateSelection', $scope.editorId, $scope.editor.selectionCss());
 
                     $scope.updateHistoryButtons();
+
+                    $scope.state = angular.copy($scope.editor.history.current());
+                    setTimeout(function() {
+                        $scope.$apply();
+                    }, 0);
```

Comment:
I need this to avoid triggering issue that digest already in progress. Without this `$apply` state is not updated